### PR TITLE
Changed some indentations in the class descritions; Copied some poten…

### PR DESCRIPTION
**Pull request by rakesh-yadav**
…tially useful functions from MagIC code's python plotting package.

Changing only the rayleigh_diagnostic file.

### CODE REVIEW

File: post_processing/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#defines a MagIC code like class for plotting
```
Comment (gassmoeller) [inline post_processing/__init__.py]: Please include the license header of the GPL into all new files (just copy from the existing files). The license requires this.

File: post_processing/plotlib.py [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+
+
+
+#-------------------------------------------------------------------
+# Content taken from 
+# https://github.com/magic-sph/magic/blob/master/python/magic/setup.py
+# File developed by Thomas Gastine (2018)
```
Comment (gassmoeller) [inline post_processing/plotlib.py]: This seems duplicative to the comment above. Maybe just use one of the two.

File: post_processing/plotlib.py [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+    else:
+        if tit and label is not None:
+            if cbar:
+                fig = plt.figure(figsize=(6,5.5))
+                ax = fig.add_axes([0.01, 0.01, 0.82, 0.9])
+            else:
+                fig = plt.figure(figsize=(5,5.5))
+                ax = fig.add_axes([0.01, 0.01, 0.98, 0.9])
+            #ax.set_title('%s: r/ro = %.3f' % (label, rad),
+            #             fontsize=24)
```
Comment (gassmoeller) [inline post_processing/plotlib.py]: Commented code mostly creates confusion, could you remove the commented parts?

File: post_processing/plots.py [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+         label = 'Temp/entropy fluc'
+        elif quantity_code == 801:
+         label = 'Br'
+        elif quantity_code == 802:
+         label = 'Bt'
+        elif quantity_code == 803:
+         label = 'Bp'
+          
+
+        #label = label + 'at radius' + '{:+f}'.format(ss.radius[rindex])
```
Comment (gassmoeller) [inline post_processing/plots.py]: see comment above

File: post_processing/rayleigh_diagnostics.py [hunk 5]
```diff
@@ -1665,3 +1693,178 @@ def streamfunction(vr,vt,r,cost,order=0):
...
+    :type exclude: bool
+    :param colat: colatitudes (when not specified a regular grid is assumed)
+    :type colat: numpy.ndarray
+    :returns: the s derivative of the input array
+    :rtype: numpy.ndarray
+
+    Function copied from file
+    https://github.com/magic-sph/magic/blob/master/python/magic/libmagic.py
+    developed by Thomas Gastine (gastine@ipgp.fr)
+
```
Comment (gassmoeller) [inline post_processing/rayleigh_diagnostics.py]: Remove empty line, you do not have one in the other functions.

**Comment by feathern** [comment]
Hey @rakesh-yadav are you still interested in this, or would you like us to just close this pull request for now?  We've changed quite a bit of the repository this week.

**Comment by rakesh-yadav** [comment]
Hi @feathern, yeah, go ahead and close it.

**Pull request closed without merging**